examples/deployment_ec2_instance/aws_ec2.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_instance" "inst" {
+  ami                    = var.ubuntu18_ami
+  instance_type          = "t2.large"
+  vpc_security_group_ids = [aws_security_group.group.id]
+  key_name               = var.keypair
+  subnet_id              = aws_default_subnet.default.id
+}
+
+# Using the default vpc & subnet
+resource "aws_default_vpc" "default" {}
+resource "aws_default_subnet" "default" {
+
+  # Using zone A for convinience (this would render as us-east-1a, in case us-east-1 is the region variable)
```

Comment:
```suggestion
resource "aws_default_subnet" "default" {
  # Using zone A for convinience (this would render as us-east-1a, in case us-east-1 is the region variable)
```